src/CMakeLists.txt

```diff
@@ -330,59 +264,68 @@ set(src_lib_static_nupiccore_srcs
 
 set(src_lib_static_nupiccore_compile_flags
     "${src_compile_flags} -I${src_numpy_core}/include")
+=======
+    nupic/utils/StringUtils.cpp
+    nupic/utils/StringUtils.hpp
+    nupic/utils/TRandom.cpp
+    nupic/utils/TRandom.hpp
+    nupic/utils/VectorHelpers.hpp
+    nupic/utils/Watcher.cpp
+    nupic/utils/Watcher.hpp
+)
+>>>>>>> Stashed changes
```

Comment:
and here and above (merge conflict)

Reply:
>  (merge conflict)

There were several that I missed.  I got them fixed.  I just hope I do not inadvertently add back in a file that you deleted.  I had so many adds and deletions it is hard to keep track.

Reply:
Which deleted file? 
And I fixed (hope) and push the merge conflicts too :/ hope it doesn't screw up the merging for you. Please try to merge it this branch (upstream to local)